### PR TITLE
New package: qarmin.czkawka.cli version 6.1.0

### LINKED ISSUE #110314: [Package Request]: czkawka
### How can we help?

I would like someone else to build the manifest.

### Please read and ensure the following

- [X] The installer meets the above requirements

### Please provide the following information

Download Page Url: https://github.com/qarmin/czkawka/releases/tag/6.0.0
Publisher: https://github.com/qarmin
Package Name: czkawka
Description:  Multi functional app to find duplicates, empty folders, similar images etc. 
Package Version: 6.0.0
Installer URL: https://github.com/qarmin/czkawka/releases/download/6.0.0/windows_czkawka_gui2.zip

### PR DESCRIPTION
**Pull request by Dragon1573**
## Description

- Resolve #110314

## Checks

- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/129929)

### CODE REVIEW

**Comment by Dragon1573** [comment]
According to https://github.com/microsoft/winget-pkgs/issues/110314#issuecomment-1848961466, this package might not pass the Azure Pipelines and cause a [Validation-Executable-Error](https://github.com/microsoft/winget-pkgs/labels/Validation-Executable-Error).

This behaviour is expected because the CLI should be launched with subcommands. The error should be waived.

File: manifests/q/qarmin/czkawka/6.1.0/qarmin.czkawka.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created with YamlCreate.ps1 v2.2.12 $debug=NVS1.CRLF.5-1-22621-2506.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: qarmin.czkawka
+PackageVersion: 6.1.0
+InstallerType: portable
+ReleaseDate: 2023-10-15
+Installers:
+- Architecture: x64
+  InstallerUrl: http://yy-huangxr/windows_czkawka_cli.exe
```
Comment (Dragon1573) [inline manifests/q/qarmin/czkawka/6.1.0/qarmin.czkawka.installer.yaml:10]:
```suggestion
  InstallerUrl: https://github.com/qarmin/czkawka/releases/download/6.1.0/windows_czkawka_cli.exe
```

**Comment by soredake** [comment]
@Dragon1573 i think this package should be name `qarmin.czkawka.cli` as it's contains only cli version.

**Comment by stephengillie** [comment]
Pending Manual Validation.

**Review by stephengillie — APPROVED** [review]

**Pull request merged**